Documentation/how-to-build-ObjectWriter.md

```diff
@@ -0,0 +1,43 @@
+# Build ObjectWriter library #
+
+ObjWriter is based on LLVM, so it requires recent CMake and GCC/Clang to build LLVM.
+See http://llvm.org/docs/GettingStarted.html#requirements for more details.
+
+1. Clone LLVM from official LLVM mirror github git repository:
+
+    ```
+    $ git clone -b release_50 https://github.com/llvm-mirror/llvm.git
+    ```
+
+2. Apply the patch to LLVM:
+
+    ```
+    $ cd llvm
+    $ wget https://gist.githubusercontent.com/BredPet/5ba0e8cc195dc3a09d11e9cb5fd74366/raw/7de9a131a5256deaf2abfdbf1d44857a61ae1f29/llvm.patch
```

Comment:
Can we have the patch committed here as well?

Reply:
Of course, I forgot about this simple solution.